gdbgui/src/js/Breakpoints.jsx

```diff
@@ -66,6 +66,22 @@ class Breakpoint extends React.Component {
       </div>
     );
   }
+  get_num_times_hit(bkpt){
+
+    if (bkpt.times === undefined) {
+      return "" // E.g. 'bkpt' is a child breakpoint
+    }
+
+    if (bkpt.times == 0) {
+      return "breakpoint never hit"
```

Comment:
recommend returning empty so that if there is no text, that means the breakpoint was never hit. This will reduce the amount of text on the screen but still (hopefully) convey the hit count to the user.